fastmath.py

```diff
@@ -89,6 +117,60 @@ def check_fastmath(pkg_dir, pkg_name):
     return
 
 
+def check_hardcoded_flag(pkg_dir, pkg_name):
+    """
+    Check if all `fastmath` flags are set to a config variable
+
+    Parameters
+    ----------
+    pkg_dir : str
+        The path to the directory containing some .py files
+
+    pkg_name : str
+        The name of the package
+
+    Returns
+    -------
+    None
+    """
+    ignore = [("fastmath", "_add_assoc")]
+
+    hardcoded_fastmath = []  # list of njit functions with hardcoded fastmath flags
+    modules_funcs_nodes = get_func_nodes(pkg_dir)
+    for module_name, func_nodes in modules_funcs_nodes.items():
+        for node in func_nodes:
+            if (module_name, node.name) in ignore:
+                continue
+
+            njit_decorator_func = None
+            for decorator in node.decorator_list:
+                if (
+                    isinstance(decorator, ast.Call)
+                    and isinstance(decorator.func, ast.Name)
+                    and decorator.func.id == "njit"
+                ):
+                    njit_decorator_func = decorator
+                    break
+
+            if njit_decorator_func is None:
+                continue
+
+            for kwrd in njit_decorator_func.keywords:
+                if kwrd.arg == "fastmath":
+                    value = kwrd.value.value
+                    if not hasattr(value, "id") or value.id != "config":
+                        hardcoded_fastmath.append(f"{module_name}.{node.name}")
```

Comment:
After addressing my previous comment, `node.name` can be replaced with `func_name`

Reply:
Please ignore